eland/operations.py

```diff
@@ -1273,6 +1274,8 @@ def to_json(  # type: ignore
             for i, df in enumerate(
                 self.search_yield_pandas_dataframes(query_compiler=query_compiler)
             ):
+                if PANDAS_VERSION[0] == 2:
```

Comment:
To prepare for pandas 3:

```suggestion
                if PANDAS_VERSION[0] >= 2:
```